app/src/main/java/org/mozilla/fenix/utils/Undo.kt

```diff
@@ -63,6 +66,31 @@ fun CoroutineScope.allowUndo(
             snackbar.view.elevation = it
         }
 
+        val shouldUseBottomToolbar = view.context.settings().shouldUseBottomToolbar
+        val toolbarHeight = view.context.resources
+            .getDimensionPixelSize(R.dimen.browser_toolbar_height)
+
+        snackbar.view.setPadding(
+            0,
+            0,
+            0,
+            if (
+                paddedForBottomToolbar &&
```

Comment:
Won't this break other places the Undo is used? from a quick search - bookmarks, search engine list? I suppose having unnecessary padding is better than covering the toolbar? But still weird.

Reply:
Ah I see it's a param we're passing in not just the bottom toolbar setting. Okey that should be fine. We should also just generally retest the Undo in the places it's used to make sure they all still work 🆗